src/manifests/network/cilium/cilium.yaml

```diff
@@ -162,7 +154,7 @@ data:
 
   # Regular expression matching compatible Istio sidecar istio-proxy
   # container image names
-  sidecar-istio-proxy-image: "ghcr.io/hlesey/cilium/istio_proxy:1.10.3"
+  sidecar-istio-proxy-image: "cilium/istio_proxy"
```

Comment:
Why not continue using the mirrored image in order to avoid the dockerhub ratelimit?

Reply:
Good catch, yes we must use it.